src/absil/ilread.fs

```diff
@@ -1029,377 +1032,373 @@ let count c =
 #endif
         
 
-let seekReadUInt16Adv ctxt (addr: byref<int>) =  
```

Comment:
(If you write the Fable version using struct-tuples, and we can get it into a PR for visualfsharp, and we can performance test it, then we can switch to that version)